bin/deploy

```diff
@@ -10,6 +10,17 @@ if [[ $# -ne 1 ]]; then
 	echo "Please specify an environment."
 	exit 1
 else
-	bundle exec middleman build --clean --environment="$ENV"
-	bundle exec middleman s3_sync --environment="$ENV"
+  if [[ $ENV =~ pro ]]
```

Comment:
I initially had `^production$`, which matches "production" exactly, but when I realized that you could run `bin/deploy prod` or `bin/deploy productions`, I decided to just go with `pro`. If the user spells at least that much right, he will still get the confirmation question.

Reply:
If the user enters a non-valid environment (like "prod") the deploy will actually fail. This is because the production environment file contains the necessary code for deployment (like the bucket name).

Reply:
So here, we should check for the exact string "production".

Reply:
Also join if/then lines (see comment further down).